### PR TITLE
Library Project

### PR DESCRIPTION
**Pull request by hemantso**
### Library Project
- New Book button to add a new book in the library
- Toggle the button to read or unread for reading status
- delete button to delete a book
- static data of library in data,js file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# library
\ No newline at end of file
+# Library
+
+![image](./static/Screenshot.png)
+
+This is a Library Javascript project where user can dynamically add books and store in localstorage. User can toggle between read and not read button. user can delete and add a book.
+
```
Comment (christianotieno) [inline README.md:6]: ## Improve Readme

- [ ] Let's add a screenshot and instructions on how to well clone and run the project locally (setup). Let's keep it more descriptive enough ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+
+const addBookToLibrary = (title, author, pages) => {
+  const book = new Book(title, author, pages);
+  myLibrary.push(book);
+  renderTable(); // eslint-disable-line
+};
```
Comment (christianotieno) [inline js/index.js]: ## Add `read` option to the form submission.
- [ ] During book creation, kindly add the option to add a book. The form should allow the marking the `Book` status as `read`

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  renderTable();
+}
+
+function handleClick() {
+  const title = document.querySelector('#title');
+  const author = document.querySelector('#author');
+  const pages = document.querySelector('#pages');
+  if (title && author && pages) {
+    addBookToLibrary(title.value, author.value, pages.value);
+    title.value = '';
```
Comment (christianotieno) [inline js/index.js]: # Refactor this entire codebase to suit the following 👇 
- [ ] Let's keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright).

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Changes Required ♻️ 
Hello team 👋 
Good job on this project. 👍 
Kindly, let's work on the following issues before we proceed 👇  

### Happy coding 💻 

#### Christian Otieno (TSE)

**Comment by hemantso** [comment]
Required changes made. Please clone the repository and run the program. Live link online showing the HTML page only.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray: 
Good job making suggested changes. We however, need to fix a few things before we can merge this PR.
- [x] validate your form inputs. A user should not be able to create a book with empty fields. Fix that.
- [x] Store books in the localStorage in the user's browser. When a user creates a book, they should be able to view that book after reloading their browser. That should not happen. Please fix that.


### Martin Afani

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @hemant-soni-vst-au4 @Bekhzod96 :wave: 

### Thank you for making all of the requested changes from the previous TSE, but I believe those changes generated a bug:

- [x] I can't add any books to the library, I checked both in your live version and also cloned your project.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by hemantso** [comment]
there was one silly mistake. Now you can add a new book in the cloned repository.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, CONGRATULATIONS ON FIXING ALL THE COMMENTS OUTLINE BY THE PREVIOUS TSE'S :+1: I hereby approved your project :smile: 

Go ahead and merge into development :fireworks: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**